source/reference/logs.txt

```diff
@@ -0,0 +1,54 @@
+.. _mdb-shell-logs:
+
+==================
+MongoDB Shell Logs
+==================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+|mdb-shell| stores logs for each session in `ndjson
+<http://ndjson.org/>`__ format using `pino
+<https://github.com/pinojs/pino>`__. With a Session ID, you can view or
```

Comment:
[nit / wordsmith] "You can view or tail the logs for a |mdb-shell| session based on its session ID."

Reply:
updated